packages/amplify-util-mock/src/api/api.ts

```diff
@@ -87,6 +89,10 @@ export class APITest {
       await this.generateTestFrontendExports(context);
       await this.generateCode(context, appSyncConfig);
 
+      if (this.appSyncSimulator.isHttps) {
+        context.print.success('\nHTTPS enabled successfully.\n');
+      }
```

Comment:
There's a print out in line 96 below that prints full url which presumably contains protocol part.
Therefore please remove this print out as it's redundant.